index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Eze Promise</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <h1 class="logo">Welcome</h1>
+        <span class="menu">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </span>
+      </header>
+      <section class="headline">
+        <div class="intro">
+          <p>Hey there. I’m Promise <span class="title">I’m a software developer</span></p>
```

Comment:
- [ ] Kindle align the title to the left.
![Captura de pantalla_2021-09-16_09-59-12](https://user-images.githubusercontent.com/53111464/133636384-55701c34-1faa-4325-af67-150d218d1adb.png)

Reply:
-Please replace the `<p>` with `<h1>`, the title in a website has to always be an `<h1>` in order to add SEO support on your website.

Reply:
I am working with template 3 and the title is not aligned to the left. Thank you for the SEO knowledge.
![template3](https://user-images.githubusercontent.com/46839548/133656655-c1d04e2e-9db5-4357-97a2-59df900e90e9.PNG)

Reply:
Yes, you are right! just replace that `p` with a `h1` tag